modules/manage/pages/iceberg/iceberg-topics-gcp-biglake.adoc

```diff
@@ -0,0 +1,272 @@
+= Query Iceberg Topics using GCP BigLake
```

Comment:
@kbatuigas I think 'Use Iceberg Topics with GCP BigLake'

The querying is really done with BigQuery or other engines.